CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Change Log
+
+## 1.0.0
```

Comment:
although we don't prefix images with `v`, we do prefix the releases, so I think this should be `v1.0.0` for consistency